### PR TITLE
CONCF-207 Upgrade to vignette-js#2.1.1

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/CONCF-207

Upgrading to vignette-js#2.1.1 was enough to fix the issue where images in gallery were not cropped properly.

### CODE REVIEW

File: front/scripts/main/components/ImageMediaComponent.ts [hunk 1]
```diff
@@ -47,22 +47,20 @@ App.ImageMediaComponent = App.MediaComponent.extend(App.ArticleContentMixin, {
...
 			media = this.get('media');
 
 			if (media) {
-				return this.getThumbURL(
-					this.get('media').url,
-					Mercury.Modules.Thumbnailer.mode.thumbnailDown,
-					this.get('articleContent.width'),
-					this.get('computedHeight')
-				);
+				return this.getThumbURL(this.get('media').url, {
```
Comment (Warkot) [inline front/scripts/main/components/ImageMediaComponent.ts]: Why don't you use `media.url`? `media` from line 56 is only used now on line 58.

Comment (rogatty) [inline front/scripts/main/components/ImageMediaComponent.ts]: `¯\_(ツ)_/¯` fixed.

Comment (Warkot) [inline front/scripts/main/components/ImageMediaComponent.ts]: Oh, I thought there might be some Ember/Vignette magic behind it. :crystal_ball: :+1:

**Comment by Warkot** [comment]
LGTM :sheep:

**Comment by hakubo** [comment]
LGTM

**Pull request merged**